### PR TITLE
xdg-open: add page

### PR DESCRIPTION
**Pull request by goznalo-git**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

Hey, a couple pages on the xdg-open tool, quite useful to open files in graphical applications. I also added the Spanish translation.

### CODE REVIEW

File: pages.es/linux/xdg-open.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# xdg-open
+
+> Abre un archivo o URL en la aplicación predeterminada del usuario.
+
+- Abre una URL en el navegador predeterminado:
+
+`xdg-open {{pagina_web}}`
```
Comment (marchersimon) [inline pages.es/linux/xdg-open.md:7]:
```suggestion
`xdg-open {{https://www.ejemplo.es}}`
```

File: pages.es/linux/xdg-open.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Abre una image en el visor de imágenes predeterminado:
+
+`xdg-open {{archivo_imagen}}`
```
Comment (marchersimon) [inline pages.es/linux/xdg-open.md:11]:
```suggestion
`xdg-open {{ruta/al/imagen}}`
```

File: pages.es/linux/xdg-open.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Abre un PDF en el visor de PDF predeterminado:
+
+`xdg-open {{archivo_pdf}}`
```
Comment (marchersimon) [inline pages.es/linux/xdg-open.md:15]:
```suggestion
`xdg-open {{ruta/al/pdf}}`
```

File: pages/linux/xdg-open.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# xdg-open
+
+> Opens a file or URL in the user's preferred application.
+
+- Open an URL in the system's default browser:
+
+`xdg-open {{website}}`
```
Comment (marchersimon) [inline pages/linux/xdg-open.md:7]:
```suggestion
`xdg-open {{https://example.com}}`
```

File: pages/linux/xdg-open.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+- Open an image in the system's default image viewer:
+
+`xdg-open {{image_file}}`
```
Comment (marchersimon) [inline pages/linux/xdg-open.md:11]:
```suggestion
`xdg-open {{path/to/image}}`
```

File: pages/linux/xdg-open.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+- Open a PDF in the system's default PDF viewer:
+
+`xdg-open {{pdf_file}}`
```
Comment (marchersimon) [inline pages/linux/xdg-open.md:15]:
```suggestion
`xdg-open {{path/to/pdf}}`
```

**Review by marchersimon — CHANGES_REQUESTED** [review]
Thanks for the new page! 
I've just a few suggestions. Also, adding `xdg-open .` for opening the current directory in the default file manager would be great.

**Review by goznalo-git — COMMENTED** [review]
All suggestions added

**Review by marchersimon — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]
Looks good to me. I'm not sure about More information link—there ought to be one, but I can only find a die.net man page, no man7.org one 🤷🏻

File: pages/linux/xdg-open.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+- Open an image in the system's default image viewer:
+
+`xdg-open {{path/to/image}}`
+
+- Open a PDF in the system's default PDF viewer:
+
+`xdg-open {{path/to/pdf}}`
+
+- Display the command's help:
```
Comment (navarroaxel) [inline pages/linux/xdg-open.md:21]:
```suggestion
- Display help:
```

File: pages/linux/xdg-open.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# xdg-open
+
+> Opens a file or URL in the user's preferred application.
+
+- Open the current directory in the system's default file explorer:
+
+`xdg-open .`
+
+- Open an URL in the system's default browser:
```
Comment (navarroaxel) [inline pages/linux/xdg-open.md:9]: Based on `gh-gist` and other pages we can simplify this sentence:
```suggestion
- Open an URL in the default browser:
```

File: pages/linux/xdg-open.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+`xdg-open {{https://example.com}}`
+
+- Open an image in the system's default image viewer:
```
Comment (navarroaxel) [inline pages/linux/xdg-open.md:13]:
```suggestion
- Open an image in the default image viewer:
```

File: pages/linux/xdg-open.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+`xdg-open {{path/to/image}}`
+
+- Open a PDF in the system's default PDF viewer:
```
Comment (navarroaxel) [inline pages/linux/xdg-open.md:17]:
```suggestion
- Open a PDF in the default PDF viewer:
```

File: pages/linux/xdg-open.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+# xdg-open
+
+> Opens a file or URL in the user's preferred application.
+
+- Open the current directory in the system's default file explorer:
```
Comment (navarroaxel) [inline pages/linux/xdg-open.md:5]:
```suggestion
- Open the current directory in the default file explorer:
```

File: pages.es/linux/xdg-open.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+- Abre una image en el visor de imágenes predeterminado:
+
+`xdg-open {{ruta/al/imagen}}`
+
+- Abre un PDF en el visor de PDF predeterminado:
+
+`xdg-open {{ruta/al/pdf}}`
+
+- Muestra la ayuda del comando:
```
Comment (navarroaxel) [inline pages.es/linux/xdg-open.md:21]:
```suggestion
- Muestra la ayuda:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by goznalo-git — COMMENTED** [review]
All changes accepted!

**Review by marchersimon — APPROVED** [review]

File: pages/linux/xdg-open.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
+# xdg-open
+
+> Opens a file or URL in the user's preferred application.
```
Comment (navarroaxel) [inline pages/linux/xdg-open.md:3]:
```suggestion
> Opens a file or URL in the user's preferred application.
> More information: <https://man.archlinux.org/man/xdg-open.1>.
```

File: pages.es/linux/xdg-open.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
+# xdg-open
+
+> Abre un archivo o URL en la aplicación predeterminada del usuario.
```
Comment (navarroaxel) [inline pages.es/linux/xdg-open.md:3]:
```suggestion
> Abre un archivo o URL en la aplicación predeterminada del usuario.
> Más información: <https://man.archlinux.org/man/xdg-open.1>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**